modules/claims_api/spec/workers/poa_form_builder_job_spec.rb

```diff
@@ -90,5 +91,29 @@
         subject.new.perform(power_of_attorney.id)
       end
     end
+
+    context 'when signature has prefix' do
+      before do
+        Veteran::Service::Representative.new(representative_id: '67890', poa_codes: ['ABC']).save!
+        Veteran::Service::Organization.create(poa: 'ABC', name: 'Some org')
+        power_of_attorney.update(form_data: power_of_attorney.form_data.deep_merge(
+          {
+            signatures: {
+              veteran: bad_b64_image,
+              representative: bad_b64_image
+            }
+          }
+        ))
+      end
+
+      it 'generates the pdf to match example' do
+        allow_any_instance_of(ClaimsApi::PoaPdfConstructor::Organization).to receive(:construct)
```

Comment:
Can you try something for me?
Can you change this to `expect_any_instance_of`?

If I'm thinking correctly, this check at this moment isn't really doing anything.  I think it's just saying, "_if an instance receives this message and raises this exception, then cool, else, don't worry about it._".  
Whereas, in this test, you're actually feeding it bad data and you are expecting it to raise an exception.